tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -126,17 +126,20 @@ class Development(CloudGov):
 
     # https://docs.djangoproject.com/en/2.0/ref/settings/#allowed-hosts
     ALLOWED_HOSTS = ['.app.cloud.gov']
+    AV_SCAN_URL = os.getenv('AV_SCAN_URL', 'http://tanf-dev-clamav-rest.apps.internal:9000')
 
 
 class Staging(CloudGov):
     """Settings for applications deployed in the Cloud.gov staging space."""
 
+    # TODO: why not just 'appcloudgov'?
     ALLOWED_HOSTS = ['tdp-backend-staging.app.cloud.gov', 'tdp-backend-develop.app.cloud.gov']
 
     LOGIN_GOV_CLIENT_ID = os.getenv(
         'OIDC_RP_CLIENT_ID',
         'urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-staging'
     )
+    AV_SCAN_URL = os.getenv('AV_SCAN_URL', 'http://tanf-staging-clamav-rest.apps.internal:9000')
```

Comment:
Don't these need `/scan` at the end?

Reply:
I remember changing this a while back and thinking this was the solution to fix the scans. It does seems strange though. I will double check.